docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-product-feature-integration.md

```diff
@@ -707,6 +730,55 @@ class ProductStorageDependencyProvider extends SprykerProductStorageDependencyPr
 }
 ```
 
+**src/Pyz/Yves/ProductDetailPage/Theme/default/components/molecules/product-configurator/product-configurator.twig**
+
+```twig
+...
+{%- raw -%}
+{% widget 'AddToCartFormWidget' args [config, data.product, isDisabled, options] only %}
+    {% block embeddedData %}
+        ...
+        {% set merchantProductOfferWidget = findWidget('MerchantProductOfferWidget', [data.product]) %}
+
+        {% set productOffersCount = merchantProductOfferWidget ? merchantProductOfferWidget.productOffers | length : 0 %}
+        {% set isRadioButtonVisible = productOffersCount > 0  %}
+        {% set isChecked = merchantProductOfferWidget ? not merchantProductOfferWidget.productView.productOfferReference: true %}
+        {% set merchantProductWidget = findWidget('MerchantProductWidget', [
+            data.product,
+            isRadioButtonVisible,
+            isChecked
+        ]) %}
+
+        {% if merchantProductWidget %}
+            {% widget merchantProductWidget %}
+            {% endwidget %}
+        {% endif %}
+    {% endblock %}
+{% endwidget %}
+...
+{% endraw %}
+```
+
+**src/Pyz/Yves/ShopUi/Theme/default/components/molecules/product-card-item/product-card-item.twig**
```

Comment:
We do not add such things to the integration guides, project-level twig can be absolutely different from what we have in core. Please get rid of this example.